test/lint/test_runner/src/main.rs

```diff
@@ -650,6 +724,10 @@ fn main() -> ExitCode {
     };
 
     let git_root = get_git_root();
+    let commit_range = commit_range();
+    let commit_log = check_output(git().args(["log", "--no-merges", "--oneline", &commit_range]))
+        .expect("git error");
+    println!("Checking commit range ({commit_range}):\n{commit_log}\n");
```

Comment:
looks like this adds a new log added to the CI (which I think is fine)

For other reviewers this is what this [log looks like in Cirrus CI](https://cirrus-ci.com/task/4637065368829952?logs=lint#L186-L188)
```
[12:23:15.442] Checking commit range (HEAD~..HEAD):
[12:23:15.442] fa42b4aa5e lint: Call lint_commit_msg from test_runner
[12:23:15.442] faee335285 lint: Call lint_scripted_diff from test_runner
```